packages/dashboard/src/customization/widgets/constants.ts

```diff
@@ -1,3 +1,15 @@
 export const SVG_STROKE_SOLID = '1,0';
 export const SVG_STROKE_DASHED = '10,5';
 export const SVG_STROKE_DOTTED = '3,3';
+
```

Comment:
these should be defined in the 
`iot-app-kit/packages/dashboard/src/customization/widgets` folder
each subfolder has a `plugin.tsx` file which defines an `initialSize`